mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -184,3 +185,15 @@ object English : LocalizedStrings {
   override val tournamentsCreateElimDemomN = { denominator: Int -> "1 / $denominator" }
   override val tournamentsCreateElimDepthFinal = "Final"
 }
+
+fun Duration.toDurationString(): String {
+  if (this.toString().contains("d")) {
+    return "${this.toLong(DurationUnit.DAYS)} days"
+  } else if (this.toString().contains("h")) {
+    return "${this.toLong(DurationUnit.HOURS)} hours"
+  } else if (this.toString().contains("m")) {
+    return "${this.toLong(DurationUnit.MINUTES)} minutes"
+  } else {
+    return "${this.toLong(DurationUnit.SECONDS)} seconds"
+  }
+}
```

Comment:
You probably should not use the `toString()` representation of the `Duration` to transform it to some English text.What would you think of inspecting the inner fields directly ?

Reply:
Hey! Yes that was my initial approach but I couldn't access the inner fields actually. Are you aware of a way to do so?